scripts/install.sh

```diff
@@ -25,13 +25,14 @@ echo "\nWorking directory at ${tmpdir}\n"
 cd $tmpdir
 
 mkdir -p deploy
-curl https://raw.githubusercontent.com/aws/aws-app-mesh-inject/v0.1.6/deploy/inject-ns.yaml > deploy/inject-ns.yaml
-curl https://raw.githubusercontent.com/aws/aws-app-mesh-inject/v0.1.6/deploy/inject.yaml.template > deploy/inject.yaml.template
+
+curl https://raw.githubusercontent.com/aws/aws-app-mesh-inject/master/deploy/inject-ns.yaml > deploy/inject-ns.yaml
```

Comment:
this somehow bothers me. Given that we do not have good testing and rely on manual verification, is it safe to use master? If managing versions is painful, at-least a 'release' branch will be agreeable. what do you think?

Reply:
Hm, that could work, yes.  In addition we could allow passing in a variable to overwrite the branch specified here (I think there was another PR that did similar).

Reply:
Yeah, agree that it would be best to have this controlled by an envvar. You could default the envvar to master, but having the ability to specify a tag would be really useful.